sdks/go/pkg/beam/core/graph/coder/coder.go

```diff
@@ -229,6 +245,7 @@ func IsWKV(c *Coder) bool {
 
 // NewWKV returns a WindowedValue coder for the window of KV elements.
 func NewWKV(components []*Coder, w *window.Window) *Coder {
+	checkCodersNotNil(components)
```

Comment:
<!--thread_id:cc_166096978_t; commit:977a4f133a10d957b90566426581193d93041321; resolved:1-->
<!--section:context-quote-->
> **wcn3** wrote:
> Consider calling this function 'checkCoders' since it doesn't need to reveal its implementation details, and affords the possibility to verify other invariants of the coders.

<!--section:body-->
I prefer to postpone such a rename to when the implementation does something non-trivial. We should not suggest validation guarantees that are not there.

Reply:
Since it's a private function, I'm OK with deferring the rename.